sdk/core/AzureCore/AzureCore.xcodeproj/project.pbxproj

```diff
@@ -8,7 +8,6 @@
 
 /* Begin PBXBuildFile section */
```

Comment:
ResumableOperation was intended to be in AzureCore because it could be reused in contexts separate from transfer manager or storage.

Reply:
I'd be okay with keeping it there, but I saw that ResumableOperation / ResumableTransfer used a lot of Transfer-specific classes, so if we want to keep it in Core (which I think is reasonable) we probably need to decouple it.

If it's kept internal in Storage, then honestly we don't need to worry about it now - it won't be part of the public API. We can always extract it out and move it to Core later.